collector/collector-build.sh

```diff
@@ -0,0 +1,14 @@
+#collector
+#cd kindling/collector
+
+#We haven't posted any tags,so i use commitId instead
+# GitCommit=$(git describe)
+GitCommit=$(git rev-parse --short HEAD || echo unsupported)
+echo "Git commit:" $GitCommit
+
+
+# docker run -it -v $PWD:/collector kindlingproject/kindling-collector-builder bash 
+docker run -it -v $PWD:/collector kindlingproject/kindling-collector-builder bash -c 'go build -o kindling-collector -ldflags="-X 'github.com/sugary199/collector-version/core.CodeVersion=$GitCommit'"'
+
+docker build -t kindling-collector -f deploy/Dockerfile .
+
```

Comment:
This is OK if you just want to contribute for #112. But I prefer to use a Makefile to implement this feature. It will introduce a little complexity but is best practice for a golang project. Anyway, it depends on you. We have had a plan for writing a Makefile later.